src/api/app/controllers/webui/patchinfo_controller.rb

```diff
@@ -26,11 +26,7 @@ def new_patchinfo
     end
 
     read_patchinfo
-    @binaries.each do |bin|
-      if @binarylist.match(bin)
-        @binarylist.delete(bin)
-      end
-    end
+    @binaries.each { |bin| @binarylist.delete(bin) }
```

Comment:
@bbrunner says that this can happen if there already are built packages in the project.

So I'll leave this as is.